app/views/channelcomparison.php

```diff
@@ -53,3 +53,28 @@
     }
     echo "</table>\n";
 }
+
+if (empty($new_strings)) {
+    echo "<h3 id='new_strings'>No new strings have been added</h3>\n";
+} else {
+    echo "<h3 id='new_strings'>New added strings in {$locale} between {$repos_nice_names[$chan1]} and {$repos_nice_names[$chan2]}</h3>";
+    echo "\n<table class='collapsable'>" .
+         "  <tr>\n" .
+         "    <th>Key</th>\n" .
+         "    <th>Value</th>\n" .
+         "    <th>en-US value</th>" .
+         "  </tr>\n";
+
+    foreach ($new_strings as $key => $value) {
+        echo " <tr>"
+                . "   <td><span class='celltitle'>Key</span><div class='string'>" . showResults::formatEntity($key) . "</div></td>\n"
+                . "   <td><span class='celltitle'>Value</span><div class='string'>" . showResults::highlight($value) . "</div></td>\n";
```

Comment:
With highlight and hundreds of rows, we output heavy HTML that can hang the browser (happens to me), since we are not styling the special characters in this view, I think you can drop all uses of showResults::highlight() and replace them by Utils::secureText() for security.